erp-client/src/app/services/snackbar/snackbar.service.ts

```diff
@@ -23,12 +23,11 @@ export class SnackbarService {
 
   /**
    * Shows a snackbar with an error message
-   * @param {string} message - the message to display on the smackbar
+   * @param {string} message - the message to display on the snackbar
+   * @param {string} action - the text of the Action button
+   * @param {object} options - the options to include for the snackbar
    */
-  public showError(message, action = 'Okay', options = {
-    duration: 3000,
-    panelClass: [ 'snackbar-success' ]
-  }) {
+  public showError(message, action, options) {
```

Comment:
Any reason to remove the default arguments here? Not a huge deal, but it'll probably break some of my code.

Reply:
My thought process behind this was making it a little more configurable for other classes that will use it. Shouldn't break your code though, I took that into consideration. Unless it's getting used outside of award.component.ts somewhere. This is the function you use in the award.component.ts file:
`  /**
   * Shows a snackbar with an error message
   * @param {string} message - the message to display on the smackbar
   */
  showSnackbarError(message, action = 'Okay') {
    this._snackBar.showError(message, action, {
      duration: 3000,
      panelClass: [ 'snackbar-error' ],
    });
  }`

You can see that the call to showError uses the same argument signature as what was defined: (message: string, action: string, options: object)

Reply:
Oh, never mind. I suck. They were just default arguments, so the same would have been achievable either way. I'll revert this but change the default class to 'snackbar-error'.

Sorry for the confusion and good catch.

Reply:
No worries! Thanks for all the work.